python/setup.py

```diff
@@ -24,14 +24,15 @@
 __version__ = '0.2.0'
 
 PYNGRAPH_SOURCE_DIR = os.path.abspath(os.path.dirname(__file__))
+USER_HOME_DIR = os.environ.get('HOME')
```

Comment:
Instead of defining `USER_HOME_DIR` here, we should add a variable such as `DEFAULT_NGRAPH_DIST_DIR`